Data/core_rulebook/cr_abilities.lua

```diff
@@ -0,0 +1,10580 @@
+-- Converted From LST file data\pathfinder\paizo\roleplaying_game\core_rulebook\cr_abilities.lst
+-- From repository https://github.com/pcgen/pcgen at commit 11ceb52482855f2e5f0f6c108c3dc665b12af237
+SetSource({
+  SourceLong="Core Rulebook",
+  SourceShort="CR",
+  SourceWeb="http://paizo.com/store/downloads/pathfinder/pathfinderRPG/v5748btpy88yj",
+  SourceDate="2009-08",
+})
+ModifyAbility({
+  Name="Default",
+  Category="Internal",
+  AllowMultiple=false,
+  Stackable=false,
+  Visible=true,
+  Definitions={
+    {
+      Name="BastardSwordExoticUse",
+      InitialValue=0,
+    },
+    {
+      Name="BypassSizeMods",
+      InitialValue=0,
+    },
+    {
+      Name="DwarvenWaraxeExoticUse",
+      InitialValue=0,
+    },
+    {
+      Name="ActivateEnlargePerson",
+      InitialValue=0,
+    },
+    {
+      Name="ActivateReducePerson",
+      InitialValue=0,
+    },
+    {
+      Name="Pool_Traits",
+      InitialValue=0,
+    },
+    {
+      Name="IsPC",
+      InitialValue=0,
+    },
+    {
+      Name="Pool_Temp_Feat",
+      InitialValue=0,
+    },
+    {
+      Name="DisableArmorBonus",
+      InitialValue=0,
+    },
+    {
+      Name="DisableShieldBonus",
+      InitialValue=0,
+    },
+    {
+      Name="Pool_CombatFeat",
+      InitialValue=0,
+    },
+    {
+      Name="SPELLFAILURE_Total",
+      InitialValue=0,
+    },
+    {
+      Name="HasWeaponFinesseFeat",
+      InitialValue=0,
+    },
+  },
+  Bonuses={
+    {
+      Category="VAR",
+      Variables={
+        "BastardSwordExoticUse",
+      },
+      Formula="1",
+      Conditions={
+        function (character)
+          return 1 <= #filter(character.Abilities, function (ability)
+            return ability.Category == "FEAT" and (ability.Name == "Exotic Weapon Proficiency (Sword (Bastard))")
+          end)
+        end,
+      },
+    },
+    {
+      Category="VAR",
+      Variables={
+        "DwarvenWaraxeExoticUse",
+      },
+      Formula="1",
+      Conditions={
+        function (character)
+          return 1 <= #filter(character.Abilities, function (ability)
+            return ability.Category == "FEAT" and (ability.Name == "Exotic Weapon Proficiency (Waraxe (Dwarven))")
```

Comment:
This feels _really_ weird and backward... why doesn't the feat just set this variable?  Everything is Yoda defined?  This ability is zero, unless you have this feat which... just exists and has no other mechanical impact?

Reply:
Yea, I have no idea, there are some really odd things in the lst files.